src/tools/kdb/set.cpp

```diff
@@ -39,7 +39,16 @@ int SetCommand::execute (Cmdline const & cl)
 	// do not resume on any get errors
 	// otherwise the user might break
 	// the config
-	kdb.get (conf, parentKey);
+	if (cl.force)
+	{
+		kdb.get (conf, parentKey);
+	}
+	else
+	{ // if not -f, do a cascading lookup so validation is not skipped
```

Comment:
```suggestion
	{ // if not -f, do a cascading lookup so validation is not skipped. This is because the spec:/ keys will also be loaded.
```

Reply:
Being a  bit more specific here doesn't hurt. The spec:/ keys being loaded only with the cascading lookup is the one key information here.